pkg/client/events.go

```diff
@@ -20,7 +20,7 @@ func NewEventStreamer(client *Client, endpoint string) *EventStreamer {
 	streamer := &EventStreamer{
 		client:        client,
 		endpoint:      endpoint,
-		eventsChannel: make(chan v1.Event),
+		eventsChannel: make(chan v1.Event, 64),
```

Comment:
what is it for?

Reply:
This [enables buffering](https://go.dev/tour/concurrency/3), so that the `readAvailableEvents()` method's behavior that tries to read multiple events from the channel actually makes sense.

Reply:
Right. For some reason in the morning I though it has buffering already. Thanks for fixing it!